lib/awful/rules.lua

```diff
@@ -457,42 +457,39 @@ function rules.execute(c, props, callbacks)
     end
 
     -- Some properties need to be handled first. For example, many properties
-    -- depend that the client is tagged, this isn't yet the case.
+    -- require that the client is tagged, this isn't yet the case.
     for prop, handler in pairs(rules.high_priority_properties) do
         local value = props[prop]
-
         if value ~= nil then
             if type(value) == "function" then
                 value = value(c, props)
             end
-
             handler(c, value, props)
         end
-
     end
 
     -- Make sure the tag is selected before the main rules are called.
-    -- Otherwise properties like "urgent" or "focus" may fail because they
-    -- will be overiden by various callbacks.
-    -- Previously, this was done in a second client.manage callback, but caused
-    -- a race condition where the order the require() would change the output.
+    -- Otherwise properties like "urgent" or "focus" may fail (if they were
+    -- overridden by other callbacks).
+    -- Previously this was done in a second client.manage callback, but caused
+    -- a race condition where the order of modules being required would change
```

Comment:
Perhaps change `required` to `loaded` here? The old comment was a bit more precise with `require()`.